### ISSUE TITLE
A Couple of Suggestions

### ISSUE THREAD

**Issue by TornadoKegan** [JKPotato-Computer/SignMaker#11]
Hello, was wondering if we could get a couple of new shields

North Texas Toll Authority Toll Shield

Oklahoma Toll Shield (might be hard to pull off due to curvature of text) 

Texas Farm to Market Rd

it would also be cool if we could get the labels for toll pass services, TXTag, Pikepass, EZPass

Thanks for your consideration

**Comment by JKPotato-Computer** [comment]
Of course! I'll need to actually look into what shields you want but if I can pull it off I'll add it.

As for the toll passes, I'll add those when I can make the whole toll tag system since right now it doesn't really work...
